src/nvim/winfloat.h

```diff
@@ -14,6 +14,10 @@ EXTERN const char *const float_anchor_str[] INIT( = { "NW", "NE", "SW", "SE" });
 EXTERN const char *const float_relative_str[]
 INIT( = { "editor", "win", "cursor", "mouse" });
 
+/// Keep in sync with WinSplit in buffer_defs.h
+EXTERN const char *const win_split_str[] INIT( = { "left", "right",
```

Comment:
these are normal window actions. it's better not define in winfloat

Reply:
True, I was unsure about that - would `win_configs.h` be a good place to put it instead?

Reply:
It's only used in `win_config.h`, so it can be a static array in `win_config.h` as well (and even only scoped in the function that uses it).

Reply:
See #26344 for an example.

Reply:
Gotcha, thank you!